docs/home/howto/LOGGING.md

```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
 ```
 After applying this policy every new index created under this one will apply to it. There is also possibility to apply policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
+## How to export Kibana reports to csv format
+
+Since v0.10 Epiphany provide posibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
```

Comment:
```suggestion
Since v0.10 Epiphany provides possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
```